filebrowser_safe/base.py

```diff
@@ -47,7 +47,7 @@ def __len__(self):
 
     @cached_property
     def filetype(self):
-        if self.is_folder:
+        if not default_storage.type_checks_slow and self.is_folder:
```

Comment:
the `browse` view performs certain checks – which with GCS are now network calls – over and over, for each object in the directory listing. this one in particular seemed unnecessary, (especially when GCS's concept of a "folder" is so uncertain as it is).